packages/devextreme/js/__internal/scheduler/header/m_date_navigator.ts

```diff
@@ -39,8 +39,8 @@ export const getDateNavigator = (header, item) => {
 };
 
 const getPreviousButtonOptions = (header) => {
-  // @ts-expect-error text
-  const ariaMessage = messageLocalization.format('dxScheduler-navigationPrevious', getCurrentViewText(header));
+  const { currentView } = header;
+  const ariaMessage = messageLocalization.format(`dxScheduler-navigationPrevious${currentView}`);
```

Comment:
Please check that `currentView` is the right property. Users can also set custom views, so you need to make sure that when user creates custom month view, this property contains `'month'` and not custom name provided by user